src/components/card/_card.scss

```diff
@@ -0,0 +1,16 @@
+.card {
+    padding: 1rem;
+    display: grid;
+    place-items: center;
+    border-radius: 1rem;
```

Comment:
Please clean up the code here

Reply:
This section centers the content inside card, as well as styling the border's radius, I don't know what else is expected for the component

Reply:
As for the transparent feature, I'm expecting the user to use rgba() when passing the backgroundColor prop

Reply:
<img width="258" alt="image" src="https://user-images.githubusercontent.com/25376645/165220182-b224eb4f-66ed-46fe-90c4-f281b29bb689.png">

For the props, did you check the usage of `enum` ?